src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+    * **Review and approval**: [Frame.io](http://frame.io/) helps creative teams reduce the total number of revision cycles with advanced review, commenting, and annotation capabilities available across a broad set of media types.
+    * **Sharing and presentation**: Customize and share creative work from one central place.
+    * **Security and governance**: View and manage all users, workspaces, and projects in one place. Take bulk actions and export information to control how you utilize usage data.
+    * **Integrations and APIs**: Native integrations and an open API connect [Frame.io](http://frame.io/) to the systems you already use to unblock sticking points and keep work in flow.
+
+* How can I move my **Azure** **Blob** storage to [Frame.io](http://frame.io/) V4 directly?
+
+    The easiest and most efficient way to upload and download bulk content, large folder structures, and entire projects. Upload your files and folder structures directly from your desktop into your [Frame.io](http://frame.io/) projects; download the original files or choose multiple proxies. 
+
+* What is **metadata**?
+  
+  Use Metadata to tag, organize, and view creative assets based on how your team works. Tag all incoming assets with metadata to organize then group into Collections to sort and view your assets based on need (assignee, due date, approval status, etc.) for easier access and superior organization. 
+* What are **Shares**?
+   
+   Preview, edit, manage and send beautiful Shares to ensure your assets are showcased in the best light for internal and external stakeholders. Fine tune every element of your Share—from branding, to descriptions, to layout—to ensure that you are presenting your work professionally and beautifully.
+* What are **Collections** in V4?
+   
+   Collections are flexible, real-time, saved views of a selection of assets that allow users to dynamically select, filter, group, and sort their media using previously tagged metadata. Collections allow users to build meaningful, efficient workflows without the tool forcing them to work within a particular approach, process or template.
+* Is the Frame V4 API **backward compatible**?  
+    No. The Frame product released at Adobe MAX operates in fundamentally different ways and emphasizes different workflows than the legacy version. Due to these differences, the new API is not backward compatible with the Legacy Product.
+
+        The Frame.io V4 API is *not* backward-compatible with earlier versions of Frame.io APIs and, generally, cannot be used to access or update resources contained within legacy accounts as there have been significant changes to the V4 concepts and data model. As such, the URIs associated with the V4 API all include a `/v4` path prefix. However, the V4 API is still evolving quickly and it is possible that new features may occasionally warrant breaking changes.  
```

Comment:
I think this is made redundant with by the sentence that precedes it.

Reply:
Took it out.